courses/free-courses-en.md

```diff
@@ -198,6 +198,7 @@
 * [IBM AI Engineering Professional Certificate](https://www.coursera.org/professional-certificates/ai-engineer) - Romeo Kienzler, Saeed Aghabozorgi, Joseph Santarcangelo, Alex Aklson et al. (Coursera)
 * [MIT Deep Learning and Artificial Intelligence Lectures](https://deeplearning.mit.edu) - Lex Fridman, et al.
 * [MIT's Artificial Intelligence](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-034-artificial-intelligence-fall-2010/) - Prof. Patrick Henry Winston (MIT OpenCourseWare)
+* [Stanford's Machine Learning with Graphs](https://web.stanford.edu/class/cs224w/) - Prof. Jurij Leskovac (Stanford University)
```

Comment:
we don't list homorifics, such as "Prof."
we don't list author affiliations either.

Reply:
```suggestion
* [Stanford's Machine Learning with Graphs](https://web.stanford.edu/class/cs224w/) - Jurij Leskovac (Stanford University)
```